llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -210,13 +210,19 @@ PPCTargetLowering::PPCTargetLowering(const PPCTargetMachine &TM,
     setLoadExtAction(ISD::SEXTLOAD, VT, MVT::i8, Expand);
   }
 
+  setTruncStoreAction(MVT::f128, MVT::f16, Expand);
```

Comment:
nit: could you please add a comment here why we can not use PWR9 instructions to do the conversion.